src/message/WaffleResponse.js

```diff
@@ -1,110 +1,131 @@
-const { getSafe, randomWaffleColor } = require('../util/WaffleUtil');
```

Comment:
This entire code seems to have been prettified. Again, plz leave formatting changes for separate pull requests so that your main code changes can be the focus.

Reply:
Alright, sounds good!